doc/vaulted-shell.1.md

```diff
@@ -29,6 +30,9 @@ OPTIONS
   When invoked this way, credentials are sourced from default locations (e.g.
   environment, configuration files, instance profile, etc.).
 
+`--refresh`
+  Refreshes the expiration time of the vault.
```

Comment:
This makes it sound like we're specifically extending the expiration of the session, which isn't true at all. Instead, we're ignoring any existing session, generating a new session with new temporary credentials. The help text should reflect as much.